server/src/main/java/com/broadcom/lsp/cobol/service/delegates/communications/ServerCommunications.java

```diff
@@ -115,7 +118,21 @@ public void notifyThatDocumentAnalysed(String uri) {
    */
   @Override
   public void notifyThatExtensionIsUnsupported(String extension) {
-    runAsync(() -> logMessage(Error, EXTENSION_UNSUPPORTED + extension));
+    runAsync(
+        () ->
+            logMessage(
+                Error, messageService.getMessage("Communications.extUnsupported", extension)));
+  }
+
+  /**
+   * show a supplied message to the client with the supplied {@link MessageType}
+   *
+   * @param messageType {@link MessageType}
+   * @param message to be displayed at client end.
+   */
+  @Override
+  public void notifyGeneralMessage(MessageType messageType, String message) {
+    runAsync(() -> showMessage(messageType, message));
   }
```

Comment:
Where do you call this method?

Reply:
https://github.com/ap891843/che-che4z-lsp-for-cobol/blob/dd2e6e91b467b1321b59399fa98cc98f4dc05587/server/src/main/java/com/broadcom/lsp/cobol/core/messages/LocaleStoreImpl.java#L118

from the localeStore, when the user selects a locale not supported yet.